src/lib/types/Languages.d.ts

```diff
@@ -169,6 +169,10 @@ export interface LanguageKeys {
 	COMMAND_SKIP_VOTES_VOTED: string;
 	COMMAND_SKIP_VOTES_TOTAL: (amount: number, needed: number) => string;
 	COMMAND_SKIP_SUCCESS: (title: string) => string;
+	COMMAND_SUPPORT_DESCRIPTION: string;
+	COMMAND_SUPPORT_EXTENDED: string;
```

Comment:
Those two should go with the rest.

Reply:
uh... what do you mean

Reply:
With the rest of the `*_DESCRIPTION` and `*_EXTENDED` strings.